### PR TITLE
Upgrade ingress library to 0.14

### PR DESCRIPTION
**Pull request by arturo-seijas**
Upgrade the ingress library to version 0.14 to keep it up to date.

### CODE REVIEW

**Comment by gtrkiller** [comment]
Could you please add a description to the PR?

**Comment by arturo-seijas** [comment]
> Could you please add a description to the PR?

Isn't it self-explanatory?

**Comment by mthaddon** [comment]
> > Could you please add a description to the PR?
> 
> Isn't it self-explanatory?

Not necessarily. Does it fix an issue or is it just to keep up to date with upstream (note that version 14 has _just_ been published). I agree that a description is always useful.

File: lib/charms/nginx_ingress_integrator/v0/ingress.py [hunk 1]
```diff
@@ -66,22 +72,19 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 10
+LIBPATCH = 13
```
Comment (gtrkiller) [inline lib/charms/nginx_ingress_integrator/v0/ingress.py:75]: LIBPATCH should be 14 :)

**Review by gtrkiller — APPROVED** [review]
LGTM

**Review by merkata — APPROVED** [review]

**Pull request merged**